Alicerce/Sources/Logging/Providers/StringProvider.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  StringProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 03/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+final class StringProvider {
```

Comment:
What's the use case for this provider? It seems that this will grow into a gigantic `String`, which will obviously consume lot's of RAM.